modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro

```diff
@@ -359,6 +359,10 @@ definition {
 			locator1 = "ObjectPortlet#SELECT_OBJECT");
 	}
 
+	macro selectObjectLayout {
+		ObjectPortlet.viewEntryDetails(entry = "${entry}");
+	}
```

Comment:
The same thing here. You don't need to create this macro. There's a macro that makes this. You can search by _ObjectAdmin.assertObjectLayout_.

Reply:
Done !